br.com.jeanhertel.adriconf.json

```diff
@@ -0,0 +1,189 @@
+{
+   "app-id": "br.com.jeanhertel.adriconf",
+   "runtime": "org.freedesktop.Platform",
+   "runtime-version": "18.08",
+   "sdk": "org.freedesktop.Sdk",
+   "command": "adriconf",
```

Comment:
Missing `"cleanup"` look at all the extra junk in your `$builddir/files` directory like headers.

Reply:
done